planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -561,7 +583,7 @@ void BehaviorPathPlannerNode::publishModuleStatus(
       running_modules.modules.push_back(module);
     }
     if (status->module_name == "LaneChange") {
-      const auto force_approval = planner_data_->approval.is_force_approved;
+      const auto force_approval = planner_data->approval.is_force_approved;
       if (
         force_approval.module_name == "ForceLaneChange" &&
         (now - force_approval.stamp).seconds() < 0.5) {
```

Comment:
@tkimura4 
can you rebase to latest in order make sure not to overwrite previous change